.buildkite/generate-tags-pipeline.sh

```diff
@@ -58,21 +71,27 @@ ymlappend "   agents:
    commands:
    - brew uninstall ./Formula/tezos-sapling-params.rb
 
+   # Since using the tag that triggered the pipeline isn't very resilient, we use the version
+   # from the tezos-client formula, which hopefully will stay the most up-to-date.
  - label: Add Big Sur arm64 bottle hashes to formulae
    depends_on:
    - \"uninstall-tsp\"
    if: build.tag =~ /^v.*/
+   soft_fail: true # No artifacts to download if all the bottles are already built
```

Comment:
At which command this step is expected to fail if bottles were already built?

Reply:
At `buildkite-agent artifact download <...>`. All other commands should be expected to succeed if it comes to them